src/geom/PrecisionModel.cpp

```diff
@@ -85,7 +91,7 @@ PrecisionModel::PrecisionModel(Type nModelType)
     :
     modelType(nModelType),
     scale(1.0),
-    gridSize(0.0)
+    gridSize(1.0)
```

Comment:
This has no knock-on effects? Did 0.0 have a special meaning before?

Reply:
Yes, before gridsize = 0.0 indicated that the scale value should be used.  But that logic has been changed to just detect which of scale or gridsize are larger, and use that.
 
Scales/gridsizes of 0 are disallowed implicitly.  But the code handles a bad input of 0 as a no-op